elements/pfe-progress-indicator/src/pfe-progress-indicator.scss

```diff
@@ -0,0 +1,68 @@
+@import "../../pfe-sass/pfe-sass";
+
+$pfe-progress-indicator--border-light: 4px solid rgba(0, 0, 0, 0.25);
+$pfe-progress-indicator--border-dark: 4px solid rgba(0, 0, 0, 0.75);
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([pfe-indeterminate]) {
+    animation: spin 1s linear infinite;
+    -webkit-animation:spin 1s linear infinite;
```

Comment:
Don't worry about manually coding vendor prefixes - we use autoprefixer and it will add that for you.  You can confirm in the rendered .css file that this is being done correctly.